open-vulnerability-clients/src/main/java/io/github/jeremylong/openvulnerability/client/kev/KevItem.java

```diff
@@ -17,6 +17,7 @@
 package io.github.jeremylong.openvulnerability.client.kev;
 
 import com.fasterxml.jackson.annotation.JsonFormat;
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
```

Comment:
```suggestion
```

Reply:
KEV does not disallow additional properties in schema so retained them: https://www.cisa.gov/sites/default/files/feeds/known_exploited_vulnerabilities_schema.json